kin-recovery/README.md

```diff
@@ -0,0 +1,180 @@
+# Backup and Restore Standalone Module
+
+This module is an optional way to back up and/or restore your account.  
+The module wraps the Kin SDK Android import and export functionalities with a UI that includes two flows - backup and restore.  
+The UI uses a password to create a QR code, which is then used to back up the account and to restore it.
+
+It is implemented as an Android library that can be incorporated into your code.  
+This library is dependent on the kin-sdk library, and we assume that whoever needs to use it is already familiar with the kin-sdk library.
+For more details on Kin SDK, go to [kin-sdk](https://github.com/kinecosystem/kin-sdk-android/tree/master/kin-sdk).
+
+## Installation
+
+The Kin Recovery module for Android is implemented as an Android library.  
+To include the library in your project, add these two statements to your `build.gradle` files.
+
+###### Snippet: Modifying project build file
```

Comment:
why we need the "Snippet" prefix for each code block? it's obvious it's a code snippet...

Reply:
I copy it from the kin-sdk readme.
I will remove them